wasp/wasp.py

```diff
@@ -595,4 +596,41 @@ def theme(self, theme_part: str) -> int:
         idx = theme_parts.index(theme_part) * 2
         return (self._theme[idx] << 8) | self._theme[idx+1]
 
+    def store_settings(self, name, value):
+        """Stores any variable as string in a text file in a 'settings'
+        folder, making it persistent across reboots. For example: used
+        to store alarms or stopwatch start time.
+        If value is a list, the items will be separated by a ";". Hence, ";" is
+        not supposed to appear in 'value'.
+        """
+        if "settings" not in os.listdir():
+            os.mkdir("settings")
+        try:
+            with open("settings/" + name, "w") as f:
+                if isinstance(value, list):
+                    f.write(";".join([str(x) for x in value]))
+                else:
+                    f.write(str(value))
+            assert name in os.listdir("settings/")
+            return True
+        except Exception as err:
+            self.notify(watch.rtc.get_uptime_ms(), {
+                "src": "Persistent settings",
+                "title": "Storing failed",
+                "body": "Error when storing '{}': '{}'".format(name, err)})
+
+    def get_settings(self, name, delete=False):
+        if "settings" not in os.listdir():
+            os.mkdir("settings")
+        if name not in os.listdir("settings/"):
+            return None
```

Comment:
Do we need pre-checks? We could just use try/except (and return None/default on the except path)

Reply:
Added so I added try blocks. I also added a new "return None" to "set" in caes it fails.